packages/contentful-migrations/core/0000000000003-create-t-page-migration.js

```diff
@@ -0,0 +1,245 @@
+/* eslint-env node */
+const { withHelpers } = require('@jungvonmatt/contentful-migrations');
+
+module.exports = withHelpers(async (migration, _context) => {
+  const tPage = migration
+    .createContentType('t-page')
+    .name('Template: Page')
+    .description('Template for a regular web page')
+    .displayField('internal_name');
+
+  tPage
+    .createField('internal_name')
+    .name('Internal name')
+    .type('Symbol')
+    .localized(false)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('title')
+    .name('Page title')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('slug')
+    .name('Slug')
+    .type('Symbol')
+    .localized(true)
+    .required(true)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('parent_page')
+    .name('Parent page')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkContentType: ['folder', 't-page'],
+      },
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Entry');
+
+  tPage
+    .createField('seo_title')
+    .name('SEO title')
+    .type('Symbol')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('seo_description')
+    .name('SEO description')
+    .type('Symbol')
+    .localized(true)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('share_image')
+    .name('Share image')
+    .type('Link')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        linkMimetypeGroup: ['image'],
+      },
+    ])
+    .disabled(false)
+    .omitted(false)
+    .linkType('Asset');
+
+  tPage
+    .createField('no_index')
+    .name('Hide page from search engines? (noindex)')
+    .type('Boolean')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('no_follow')
+    .name('Exclude links from search rankings? (nofollow)')
+    .type('Boolean')
+    .localized(false)
+    .required(false)
+    .validations([])
+    .disabled(false)
+    .omitted(false);
+
+  tPage
+    .createField('theme')
+    .name('Theme')
+    .type('Symbol')
+    .localized(false)
+    .required(false)
+    .validations([
+      {
+        in: ['light', 'dark'],
+      },
+    ])
+    .disabled(true)
+    .omitted(true);
+
+  tPage
+    .createField('stage')
+    .name('Stage')
+    .type('Link')
+    .localized(false)
+    .required(true)
```

Comment:
Maybe we should not make the stage required?

Reply:
I find it helpful in projects that the stage is required on each side. Also for reasons of consistency. And if the m-header is fixed, for example, then there might be a display error if m-text is placed at the top. And it also helps for SEO ensuring that there is always an h1 on every page. And maybe performance... you could simply use the m-header and m-stage styles for critical css (because you know that both are always on top) and the following modules are loaded by file.